docs/docs/client.md

```diff
@@ -92,76 +127,77 @@ any other http4s backend, it presents the exact same `Client`
 ```
 
-Note that we don't have any output yet.  We have a `IO[String]`, to
-represent the asynchronous nature of a client request.
-
-Furthermore, we haven't even executed the request yet.  A significant
-difference between a `IO` and a `scala.concurrent.Future` is that a
-`Future` starts running immediately on its implicit execution context,
-whereas a `IO` runs when it's told.  Executing a request is an
-example of a side effect.  In functional programming, we prefer to
-build a description of the program we're going to run, and defer its
-side effects to the end.
+We don't have any output from the server yet as we have not executed the
+request.  We have an `IO[String]` value, which is a description of a
+program that will request a `String` from the server when run.
 
-Let's describe how we're going to greet a collection of people in
-parallel:
+Let's build another program that makes requests in parallel to greet a
+collection of people:
 
 ```scala mdoc:silent
-import cats._, cats.effect._, cats.implicits._
+import cats.effect.IO
+import cats.syntax.all._
 import org.http4s.Uri
-```
 
-```scala mdoc:silent
 def hello(name: String): IO[String] = {
   val target = uri"http://localhost:8080/hello/" / name
   httpClient.expect[String](target)
 }
 
-val people = Vector("Michael", "Jessica", "Ashley", "Christopher")
+val names = List("Ember", "http4s", "Scala")
 
-val greetingList = people.parTraverse(hello)
+val greetingList: IO[List[String]] =
+  names.parTraverse(hello)
 ```
 
-Observe how simply we could combine a single `F[String]` returned
-by `hello` into a scatter-gather to return a `F[List[String]]`.
+We use `parTraverse` to apply `hello` to each name and collect the results into
+one `IO[List[String]]`. The `par` prefix on `parTraverse` indicates that this
+will happen concurrently not sequentially.
 
-## Making the call
+### Running a Request
 
-It is best to run your `F` "at the end of the world."  The "end of
-the world" varies by context:
+We have built two programs: `helloEmber` will make a single request to get a greeting for Ember,
+and `greetingList` will make multiple concurrent requests getting multiple greetings.
+In a production application we would likely compose these programs with other programs
+up until we finally pass them to `run` in `IOApp` as seen in our intro example.
 
-* In a command line app, it's your main method.
-* In an `HttpApp[F]`, an `F[Response[F]]` is returned to be run by the
-  server.
-* Here in the REPL, the last line is the end of the world.  Here we go:
+Here in [mdoc], or in a REPL, we manually run the `IO` with `unsafeRunSync()`.
+Remember, you should not do this in your applications.
 
 ```scala mdoc
+helloEmber.unsafeRunSync()
+
 greetingList.map(_.mkString("\n")).unsafeRunSync()
 ```
 
+
 ## Constructing a URI
 
-Before you can make a call, you'll need a `Uri` to represent the endpoint you
+Before you can make a Request, you'll need a `Uri` to represent the endpoint you
```

Comment:
I wonder if we should say "run" a request, since that is what the method on `Client` is called. also nit, lowercase request or uppercase `Request` ?

Reply:
Also, "before" ... should this section have come before? We kind of cheated with the `expect` method which just takes a `String`. I wonder if we should change that in http4s :)